examples/devtools/index.js

```diff
@@ -79,7 +79,7 @@ const Application = () => (
   </container>
 );
 
-ReactHardware.render(
+render(
   <Application />,
```

Comment:
I'd like to keep the `render` callsites as is. This is because when folks look at example I want this to be very clear that render is from the ReactHardware branch.

A future goal of the project is to have all of this work with a DOM/visual renderer for creating prototypes or other visualized/emulated representations of the physical hardware.